Pipeline/PackageBuildOutputs/utilities/WaziDeployManifestGenerator.groovy

```diff
@@ -0,0 +1,178 @@
+import groovy.transform.*
+import groovy.yaml.YamlBuilder
+import com.ibm.dbb.build.report.records.*
+
+/*
+ * This is a utility method to generate the Wazi Deploy Application Manifest file  
+ */
+
+/**
+ * Initialize application manifest file
+ * Should be the constructor
+ */
+
+@Field WaziDeployManifest wdManifest = new WaziDeployManifest()
+
+def initWaziDeployManifestGenerator(Properties props) {
+	ArrayList<Artifact> artifacts = new ArrayList()
```

Comment:
Any reason to allocate a separate object here, before doing the assignment later?
Could it be done on one line like `wdManifest.artifacts = new ArrayList<Artifact>()`?